src/core/helpers.js

```diff
@@ -1250,6 +1250,41 @@ wb.escapeAttribute = function( str ) {
 	return str.replace( /'/g, "&#39;" ).replace( /"/g, "&#34;" );
 };
 
+/*
+* Find most common Personal Identifiable Information (PII) in a string and return either the cleaned string either true/false
+* @param {string} str
+* @param {boolean} toClean
+* @return {string | true | false}
+* @example
+* wb.findCommonPII( "email:test@test.com, phone:123 123 1234", true )
+* returns "email:, phone:",
+* wb.findCommonPII( "email:test@test.com, phone:123 123 1234", false )
+* returns true
+*/
+wb.findCommonPII = function( str, toClean ) {
+
+	if ( typeof str  !== "string" ) {
+		return false;
+	}
+	var regEx = [
+			/\b(?:\w{0,2}[\s\\.-]*?\d[\s\\.-]*?){5,}\b/ig, //5digits or more pattern
```

Comment:
This regex may return false positive.
We should have specific regex for specific number patterns 
The other thing if letters and numbers are submitted together the numbers won't be detected (i.e. tel5142825555)

CC @HamzaAburaneh

Reply:
@polmih can you adapt the regex to support the test case of delisma where it don't need start with a space.

Reply:
ex: `\b(?:\w{0,}[\s\\.-]*?\d[\s\\.-]*?){5,}\b`

Reply:
changed. It will take in consideration any word characters 0 or multiple times at the beginning of the 5 digits